src/set.rs

```diff
@@ -145,10 +146,11 @@ impl<T, S> OrderSet<T, S>
         self.map.reserve(additional);
     }
 
-    /// Insert the value into the map.
+    /// Insert the value into the set.
     ///
-    /// If the value did already exist in the set, returns `false`;
-    /// otherwise, return `true`.
+    /// If the value did already exist in the set, return `false`
+    /// without altering its insertion order; otherwise, insert it and
```

Comment:
Please mention that the old value remains and that the new is not used/not inserted.

Maybe just -- *if an equivalent value already existed in the set, it remains and the method returns false.*  This covers both parts

Reply:
Ok, maybe mention something about order. It remains in the set, in its place in the order. If you want to.